pytorch_lightning/trainer/training_loop.py

```diff
@@ -169,22 +170,22 @@ class TrainerTrainLoopMixin(ABC):
     def __init__(self):
         # this is just a summary on variables used in this abstract class,
         #  the proper values/initialisation should be done in child class
-        self.max_nb_epochs = None
+        self.max_epochs = None
+        self.min_epochs = None
```

Comment:
is the min needed here in this class?

Reply:
yes, it is used in the training loop.

Reply:
`self.min_nb_epochs` was there before, just lower down in the list, so I moved it up so it would be next to `self.max_epochs`.